services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/store/model/JdbcBasedDeviceDto.java

```diff
@@ -77,11 +81,18 @@ public static JdbcBasedDeviceDto forRead(final String tenantId, final String dev
                 device,
                 recordJson.getBoolean("auto_provisioned"),
                 recordJson.getBoolean("auto_provisioning_notification_sent"),
-                recordJson.getInstant("created"),
-                recordJson.getInstant("updated_on"),
```

Comment:
The `JsonObject#getInstant` method is also used in `MongoDbBasedTenantDao` and `FileBasedRegistrationService` in a similar fashion. Should it be replaced there as well?

Reply:
The problem with the JDBC registry has been introduced by https://github.com/vert-x3/vertx-jdbc-client/issues/231 with vert.x 4.2.0. The strings contained in the *created* and *updated_on* properties are not ISO datetime with timezone strings and therefore need to be handled explicitly as UTC timestamps. In the Mongo DB and the file based DB the instants are stored as proper ISO datetime with timezone strings. Thus, no special handling is required there.